README.md

```diff
@@ -12,8 +12,13 @@ If you would like to read about the documentation, just check out the [Clarkson]
 
 ## What do we use
 
+### Required
 - Clarkson Core [(A WordPress Twig plugin)](http://wp-clarkson.com/core)
+
+### Optional
+Although you don't need to use these parts, it makes the developement proces a bit smoother.
+
 - Clarkson Hooks [WordPress Hooks and Autoloading Namespaced Classes](https://github.com/level-level/clarkson-hooks)
```

Comment:
Is not optional if it is a require in composer.json

Reply:
ok